traffic_ops/traffic_ops_golang/origin/origins.go

```diff
@@ -487,3 +488,217 @@ func deleteQuery() string {
 WHERE id=:id`
 	return query
 }
+
+// Get is the handler for GET requests to Origins of APIv5
+func Get(w http.ResponseWriter, r *http.Request) {
+
+	var useIMS bool
+
+	inf, sysErr, userErr, errCode := api.NewInfo(r, nil, nil)
+	if sysErr != nil || userErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	origins, userErr, sysErr, errCode, maxTime := getOrigins(w.Header(), inf.Params, inf.Tx, inf.User, useIMS)
+	var returnable []tc.OriginV5
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+
+	for _, origin := range origins {
+		returnable = append(returnable, origin.ToOriginV5())
+	}
+	_ = maxTime // consider removing maxTime if it's not used in getOrigins in future
+
+	api.WriteResp(w, r, returnable)
+	return
+}
+
+// Create Origin with the passed data for APIv5.
+func Create(w http.ResponseWriter, r *http.Request) {
+	inf, sysError, userError, errorCode := api.NewInfo(r, nil, nil)
+	tx := inf.Tx
+	if sysError != nil || userError != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errorCode, userError, sysError)
+		return
+	}
+	defer inf.Close()
+	org, readValErr := readAndValidateJsonStruct(r)
+	if readValErr != nil {
+		api.HandleErr(w, r, tx.Tx, http.StatusBadRequest, readValErr, nil)
+		return
+	}
+
+	userErr, sysErr, errCode := checkTenancy(org.TenantID, org.DeliveryServiceID, tx, inf.User)
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+
+	_, cdnName, _, err := dbhelpers.GetDSNameAndCDNFromID(inf.Tx.Tx, *org.DeliveryServiceID)
+	if err != nil {
+		api.HandleErr(w, r, tx.Tx, http.StatusInternalServerError, fmt.Errorf("Error checking the database for delivery service name and cdn, whether it existed, and any error"), nil)
```

Comment:
error messages should not begin with a capitalized word as though they were the beginning of a sentence, and they really shouldn't start with the word "error" at all, since that's implied by the fact that it's literally an `error`. Also, this message tells me nothing about what actually happened; all of that information is carried by `err`, which you aren't using.